packages/xpetra/src/CrsMatrix/Xpetra_TpetraCrsMatrix.hpp

```diff
@@ -449,8 +450,8 @@ namespace Xpetra {
 
     //! Deep copy constructor
     TpetraCrsMatrix(const TpetraCrsMatrix& matrix)
-      : mtx_ (matrix.mtx_->template clone<Node> (matrix.mtx_->getNode ())) {}
-
+      :  mtx_(Teuchos::rcp(new Tpetra::CrsMatrix<Scalar,LocalOrdinal,GlobalOrdinal,Node>(*(matrix.mtx_),Teuchos::Copy))) {}
```

Comment:
I know `clone` worked for matrices that weren't fill complete; will the matrix always be fill complete at this point?